pages/agenda.tsx

```diff
@@ -53,6 +54,14 @@ export default function Agenda() {
           </div>
         </div>
 
+        {new Date().getTime() > 1709798400000 && 
+        <div className="container-stream-link">
+          <h2>We are live!</h2>
+          <a className="button" href={YT_LIVE_LINKS[trackSelected.toLowerCase()]} rel="nofollow" target="_blank">
```

Comment:
```
4:17:35 PM: 60:11  Error: Using target="_blank" without rel="noreferrer" (which implies rel="noopener") is a security risk in older browsers: see https://mathiasbynens.github.io/rel-noopener/#recommendations  react/jsx-no-target-blank
```

Maybe without the `rel=...`???
```suggestion
          <a className="button" href={YT_LIVE_LINKS[trackSelected.toLowerCase()]} target="_blank">
```